core/language/en-GB/ThemeTweaks.multids

```diff
@@ -26,17 +28,34 @@ Metrics/FontSize: Font size
 Metrics/LineHeight: Line height
 Metrics/BodyFontSize: Font size for tiddler body
 Metrics/BodyLineHeight: Line height for tiddler body
+Metrics/PreviewSliderWidth: Preview slider width
+Metrics/PreviewSliderWidth/Hint: the width of the slider between editor and editor preview
```

Comment:
Are those hints within a sentence? Otherwise the sentences should start with capital letters